Fixed dependencies in 'debian/control'

hwittenborn opened the pull request.

hwittenborn committed:
Fixed dependencies in 'debian/control'

gamer4life1 commented:
https://github.com/makedeb/makedeb/runs/4501675376?check_suite_focus=true#step:7:110
Seems to be invalid

hwittenborn committed:
Merge branch 'alpha' into dep-fix

lpuv reviewed (approved):
LGTM except for a issue, but I trust you can fix

hwittenborn committed:
Moved back to actions/checkout to clone the Git repository

Looks like the Git repository didn't clone previously due to us not having Git installed.

hwittenborn committed:
Fixed dependencies in debian/control and PKGBUILD/pkgbuild.sh

hwittenborn committed:
Bumped package version
